addon/components/docs-logo/index.js

```diff
@@ -35,8 +35,8 @@ export default class DocsLogo extends Component {
   @equal('logo', 'ember-data')
   showEmberData;
 
-  didReceiveAttrs() {
-    super.didReceiveAttrs(...arguments);
+  constructor() {
```

Comment:
@rwwagner90 Just realized that this won't throw if the user changes the argument to an invalid logo. Only throws on first render.

I think the chances of someone doing that are pretty slim though. But if you think it's worth it, I can have this assertion on a `did-update` render modifier.

Reply:
@miguelcobain no, I think just throwing initially is fine